.gitignore

```diff
@@ -4,7 +4,8 @@ bin
 .golangci-bin
 .coverage/
 .trivy-bin
-
+coverage/
+coverage-without-sigint/
```

Comment:
We already added .coverage to this file, what these two folders for?
Please don't update it if they are just your local test folders.

Reply:
Sorry, didn't mean to commit this.